### DISCUSSION TITLE
Add *Block Preview* to the Paste preview feature!

### DISCUSSION THREAD

**Discussion by CaptainCooky** [Mojang/minecraft-editor#53] [Feedback]
While the *Paste Preview* (Ctrl+Shift+V) is a great feature, it is hard to visualise when copying/pasting irregular shapes as there is no preview of the actual blocks, only the selection box itself. 
We could work around this by individually selecting the blocks only and not any adjacent air blocks; however, this is often inefficient and time-consuming if it's not required.  As it currently sits, rotating and flipping the selection is difficult when you can't see an actual preview of the Blocks to be pasted.

**Comment by CraftyAdventurer** [comment]
I agree but they pretty much confirmed it [here](https://github.com/Mojang/minecraft-editor/discussions/2), under _Known issues_, bullet 11:
> Paste Preview is currently just an empty selection outline. We will be iterating on this to contain an actual preview of the contents you want to paste, and also testing out attaching it to the cursor for more precise placement.

**Reply by CaptainCooky** [reply]
Ah, Thanks; I did check the Known Issues, I must've skipped past that one. Good to know it's planned already.

**Comment by minecraftchloe** [comment]
Heyo!! We're on the same page ❤️ Definitely want to improve the visualizations here! Thank you so much for writing up your feedback!
